lib/fog/openstack/volume.rb

```diff
@@ -17,12 +18,12 @@ class Volume < Fog::Service
                       :openstack_project_domain_id, :openstack_user_domain_id, :openstack_domain_id,
                       :openstack_identity_api_version]
 
-      # Fog::OpenStack::Image.new() will return a Fog::OpenStack::Volume::V2 or a Fog::OpenStack::Volume::V1,
-      #  choosing the V2 by default, as V1 is deprecated since OpenStack Juno
+      # Fog::OpenStack::Image.new() will return a Fog::OpenStack::Volume::V3 or a Fog::OpenStack::Volume::V2 or a Fog::OpenStack::Volume::V1,
+      #  choosing the V3 by default, as V1 is deprecated since OpenStack Wallaby
       def self.new(args = {})
         @openstack_auth_uri = URI.parse(args[:openstack_auth_url]) if args[:openstack_auth_url]
         service = if inspect == 'Fog::OpenStack::Volume'
-                    Fog::OpenStack::Volume::V2.new(args) || Fog::OpenStack::Volume::V1.new(args)
+                    Fog::OpenStack::Volume::V3.new(args) || Fog::OpenStack::Volume::V2.new(args) || Fog::OpenStack::Volume::V1.new(args)
```

Comment:
SInce Pike, we should use the version 3: "The Cinder v2 API has now been marked as deprecated. All new client code should use the v3 API. API v3 adds support for microversioned API calls. If no microversion is requested, the base 3.0 version for the v3 API is identical to v2."  
source: https://docs.openstack.org/releasenotes/cinder/pike.html

Reply:
I made some propositions to fix rubocop offenses and a lack of documentation in /supported.md